### PR TITLE
Test mapping of DatabaseAlreadyExists on postgres

### PR DESCRIPTION
**Pull request by tomhoule**

### CODE REVIEW

File: src/connector/postgres.rs [hunk 1]
```diff
@@ -432,7 +432,7 @@ impl PostgreSql {
 
         let schema = url.schema();
 
-        // SET NAMES sets the client text encoding. It needs to be explicitly set for automatic
+        // SET NAMES sets the client text encoding. It needs to be explicitly set for automatic
```
Comment (tomhoule) [inline src/connector/postgres.rs:435]: non-breaking space -> regular space

File: src/connector/postgres.rs [hunk 2]
```diff
@@ -700,6 +700,24 @@ mod tests {
         assert!(matches!(err.kind(), ErrorKind::AuthenticationFailed { user } if user == "WRONG"));
     }
 
+    #[tokio::test]
```
Comment (pimeys) [inline src/connector/postgres.rs:703]: Could this be actually tested in the integration tests for all (or most) databases?

**Review by pimeys — CHANGES_REQUESTED** [review]

File: src/connector/postgres.rs (hunk 2 above)
Comment (pimeys) [reply src/connector/postgres.rs:703]: Meaning, in here https://github.com/prisma/quaint/blob/master/src/tests/query/error.rs

**Comment by tomhoule** [comment]
This PR is stale, closing.

**Pull request closed without merging**